<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Фильтр импорта тарифов для Архангельской области</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="14" column="0">
    <widget class="QPushButton" name="btnAdd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="cmbMES">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Нет или не выполнен</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Выполнен</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblMES">
     <property name="text">
      <string>МЭС</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>368</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="18" column="3">
    <widget class="QPushButton" name="btnContinue">
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>Тип события</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblInfis">
     <property name="text">
      <string>Код ИНФИС</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblTariffType">
     <property name="text">
      <string>Тарификация</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="edtInfis"/>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="cmbTariffType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbEventType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbDepartment"/>
   </item>
   <item row="17" column="0" colspan="4">
    <widget class="QListWidget" name="lstService"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblDepartment">
     <property name="text">
      <string>Группа услуг</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblAmount">
     <property name="text">
      <string>Количество</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QSpinBox" name="spbAmount">
     <property name="minimum">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLabel" name="lblGroup">
     <property name="text">
      <string>Группа</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QLineEdit" name="edtGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QPushButton" name="btnDelete">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="chkSpecOrgStructure">
     <property name="text">
      <string>Подгрузить тариф для выделенного подразделения</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QCheckBox" name="chkIncompleteCaseStomotology">
     <property name="text">
      <string>Подгрузить тариф для незаконченного случая(только для стоматологии)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chkIsStac">
     <property name="text">
      <string>Подгрузить тарифы для стационара/дненовго стационара</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnContinue</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnContinue</sender>
   <signal>clicked()</signal>
   <receiver>FilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
